utilities/mtv_migration.py

```
@@ -143,6 +143,7 @@ def create_plan_resource(
     target_affinity: dict[str, Any] | None = None,
     vm_target_namespace: str | None = None,
     migrate_shared_disks: bool | None = None,
+    xfs_compatibility: bool = False,
```

Ruff (0.15.13)

[warning] Boolean-typed positional argument in function definition

(FBT001)